metricbeat/module/kubernetes/_meta/kibana/7/lens/kubernetes-32e1cac0-bcba-11ec-b64f-7dd6e8e82013.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
So all dashboards created through lens initially and then exported here correct?
That is why we have this folder?